phpspy.c

```diff
@@ -266,13 +266,17 @@ static int main_fork(int argc, char **argv) {
     if (fork_pid == 0) {
         redirect_child_stdio(STDOUT_FILENO, opt_path_child_out);
         redirect_child_stdio(STDERR_FILENO, opt_path_child_err);
+        ptrace(PTRACE_TRACEME, 0, NULL, NULL);
         execvp(argv[optind], argv + optind);
         perror("execvp");
         exit(1);
     } else if (fork_pid < 0) {
         perror("fork");
         exit(1);
     }
+    wait(NULL);
+    ptrace(PTRACE_EVENT_EXEC, fork_pid, NULL, NULL);
+    ptrace(PTRACE_CONT, fork_pid, NULL, NULL);
```

Comment:
this happens to work right now but I should probably put this in a while loop and break on `PTRACE_EVENT_EXEC` for good measure